website/integrations/services/phpipam/index.md

```diff
@@ -0,0 +1,214 @@
+-   `guest-permission-group` in place of your company naming convention
+
+:::note
+This is based on authentik 2023.3.1 and phpIPAM 1.5.2
+Only settings that have been modified from default have been listed.
+:::
+
+## authentik Configuration
+
+### Step 1 - User and Group creation
+
+You need to ensure users and groups exist before we proceed with the next steps.
+
+The groups are used for property mappings later to give the user the correct permission level in the application. For this documentation there is an example for each of the 3 main default permission levels and an easy way to visualise the differences between them.
+
+1. **Create test-user 0**
+
+    - username: test-user0
+    - Name: Test User0
+    - Email: test-user0@domain.company
+    - path: users
+
+2. **Create test-user 1**
+
+    - username: test-user1
+    - Name: Test User1
+    - Email: test-user1@domain.company
+    - path: users
+
+3. **Create test-user 2**
+
+    - username: test-user2
+    - Name: Test User2
+    - Email: test-user2@domain.company
+    - path: users
+
+4. **Create all required groups (under _Directory/Groups_)**
+
+    - admin-permission-group
+    - operator-permission-group
+    - guest-permission-group
+
+5. **Assign groups to users for testing**
+
+    - admin-permission-group
+        - Select Directory => Groups
+        - Select `admin-permission-group`
+        - Select Add Existing user
+        - Select test-user0
+    - operator-permission-group
+        - Select Directory => Groups
+        - Select `operator-permission-group`
+        - Select Add Existing user
+        - Select test-user1
+    - guest-permission-group
+        - Select Directory => Groups
+        - Select `guest-permission-group`
+        - Select Add Existing user
+        - Select test-user2
+
+### Step 2 - Property Mapping creation
+
+In order to support automatic user provisioning (JIT) with phpIPAM, additional SAML attributes need to be passed. See [phpipam docs](https://github.com/phpipam/phpipam/blob/master/doc/Authentication/SAML2.md#automatic-user-jit-provisioning) for more details about specific attributes to pass.
+
+-   Select Property Mappings
+-   Select Create => SAML Property Mapping => Next
+
+1. display_name
+
+    - Name: phpipam-display-name
+    - SAML Attribute Name: display_name
+    - Expression:
+
+    ```python
+    return user.name
+    ```
+
+2. email
+
+    - Name: phpipam-email
+    - SAML Attribute Name: email
+    - Expression:
+
+    ```python
+    return user.email
+    ```
+
+3. is_admin
+
+    - Name: phpipam-is-admin
+    - SAML Attribute Name: is_admin
+    - Expression:
+
+    ```python
+    return ak_is_group_member(request.user, name="admin-permission-group")
+    ```
+
+4. groups
+
+    - Name: phpipam-groups
+    - SAML Attribute Name: groups
+    - Expression:
+
+    ```python
+    if  ak_is_group_member(request.user, name="operator-permission-group"):
```

Comment:
```suggestion
    if ak_is_group_member(request.user, name="operator-permission-group"):
```

Reply:
Fixed this as well now. Extra space was also in my provider config, so fixed that locally too! Thanks for the spot.